plugins/jobs/src/js/types/JobRunStatusSummary.ts

```diff
@@ -19,77 +19,14 @@ export function JobRunStatusSummaryTypeResolver(
   history: MetronomeJobHistorySummary
 ): JobRunStatusSummary {
   const summary = JobHistorySummaryTypeResolver(history);
-  let status = "N/A" as JobRunStatus;
-  let time: string | null = null;
-  let lastFailureAt = 0;
-  let lastSuccessAt = 0;
-
-  if (summary.lastFailureAt !== null) {
-    lastFailureAt = DateUtil.strToMs(summary.lastFailureAt);
-  }
-
-  if (summary.lastSuccessAt !== null) {
-    lastSuccessAt = DateUtil.strToMs(summary.lastSuccessAt);
-  }
-
-  if (summary.lastFailureAt !== null || summary.lastSuccessAt !== null) {
-    if (lastFailureAt > lastSuccessAt) {
-      status = "Failed" as JobRunStatus;
-      time = summary.lastFailureAt;
-    } else {
-      status = "Success" as JobRunStatus;
-      time = summary.lastSuccessAt;
-    }
+  if ([summary.lastFailureAt, summary.lastSuccessAt].every(x => x === null)) {
```

Comment:
We could avoid an array and a function call with just `summary.lastFailureAt  === null && summary.lastSuccessAt === null` curious what's the value add?

Reply:
i'm just notoriously bad at parsing long boolean statements in my head and that's one way to directly see "every(thing) is null". 

as that's not a long statement, i'll happily change that to your suggestion!